controllers/storeController.js

```diff
@@ -115,23 +142,21 @@ const deleteStore = async (req, res) => {
 
         if (!store) {
             return res.status(404).json({
```

Comment:
Ini pake new ApiError aja bang, contohnya kayak di bawah
![image](https://github.com/akbarrahmatm/travel-be-project-team-5/assets/88664674/83d0a9c7-1853-4120-9ce8-d9100a564b35)